client/components/jetpack/server-credentials-wizard-dialog/index.tsx

```diff
@@ -96,7 +97,6 @@ const ServerCredentialsWizardDialog = ( {
 						{ translate( 'Need help? Find your server credentials' ) }
 					</ExternalLink>
 					<ServerCredentialsForm
-						className="server-credentials-wizard-dialog__form"
```

Comment:
Does this classname need to be added back somewhere else?

Reply:
I don't know, but it's not doing anything currently so I figure that it's not required.

Reply:
(It's also not styled anywhere that I can find.)